browser/src/ClinvarVariantsTrack/ClinvarVariantTrack.tsx

```diff
@@ -211,14 +220,28 @@ const ClinvarVariantTrack = ({ referenceGenome, transcripts, variants }: Props)
               {isExpanded ? 'Collapse to bins' : 'Expand to all variants'}
             </Button>
           </ControlRow>
-          <ControlRow>
+          <FilterRow>
             <Checkbox
               id="clinvar-track-in-gnomad"
               label="Only show ClinVar variants that are in gnomAD"
               checked={showOnlyGnomad}
               onChange={setShowOnlyGnomad}
             />
-          </ControlRow>
+            <label htmlFor="star-filtering">
+              Filter by Stars: &nbsp;
```

Comment:
Maybe this should be "Filter by review status"?

See: https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/

![image](https://github.com/broadinstitute/gnomad-browser/assets/8867295/c7d6d46d-bb7b-436f-baa0-b0e92a82452c)

Should we also link to this page in the label so that users can understand what review status means? I.e. the label becomes "Filter by <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/">review status</a>" or whatever the appropriate link component is.